src/PhpSpreadsheet/Calculation/LookupRef/ExcelMatch.php

```diff
@@ -87,32 +87,61 @@ public static function MATCH($lookupValue, $lookupArray, $matchType = self::MATC
         return ExcelError::NA();
     }
 
-    private static function matchFirstValue($lookupArray, $lookupValue)
+    /**
+     * @param mixed $lookupValue
+     *
+     * @return mixed
+     */
+    private static function matchFirstValue(array $lookupArray, $lookupValue)
     {
-        $wildcardLookup = ((bool) preg_match('/([\?\*])/', $lookupValue));
-        $wildcard = WildcardMatch::wildcard($lookupValue);
+        if (is_string($lookupValue)) {
```

Comment:
Do we need an is not numeric check here?

Reply:
I don't think so, although this is another confusing aspect of MATCH. If you look at the test to which I added the label 'string compared to number type -1', numeric 6 does not seem to match string '6', and that test returned `#N/A` both before and after this change, as does Excel. I will add more tests.